### ISSUE TITLE
Test lowered minimum donation

### ISSUE THREAD

**Issue by roseregina** [MozillaFoundation/Advocacy#688]
We could try out different minimum donation amounts and see it's effect on conversion and fraud. 
Currently, minimum is $2, which has the benefit of meaning that any email that is talking about $1 is about a specific stripe test charge. We also need to consider the effect that it may have on fraud rates. 

*Start date: December 21, 2016* 
*Due date: September 2, 2018* 

CC: @stephaniemcv @ScottDowne @cadecairos @vojtechmzl

**Comment by ScottDowne** [comment]
Yeah, I agree with challenging the 2$ min amount if we are getting complaints. We have room to wiggle here when it comes to min amounts, so let's figure out what converts best.

Also just a reminder that we need numbers roughly equal to 2$ in each currency.

**Comment by roseregina** [comment]
We have gotten a number of under $2 donations in other currencies, which I now assume are due to currency fluctuations. Should I be noting them for updating anything under say $1.75 USD?

example: 
![image](https://cloud.githubusercontent.com/assets/15040203/21534281/d019cc66-cd32-11e6-8d70-5d3696ba5491.png)

**Comment by ScottDowne** [comment]
Yeah, conversion rates fluctuate more than we can keep up to date, and it's not super important that we're exact, so it's just close.

I'm pretty sure fees are based on the native currency anyway, and MXN lowest is 30 MXN, which is always going to result in a profit regardless of what it is in USD. (1.42 USD is still acceptable anyway)
